executor/src/witgen/jit/variable.rs

```diff
@@ -16,6 +16,8 @@ pub enum Variable {
     /// An input or output value of a machine call on a certain
     /// identity on a certain row offset.
     MachineCallParam(MachineCallVariable),
+    /// A fixed column cell.
+    FixedColumn(Cell),
```

Comment:
```suggestion
    FixedCell(Cell),
```
No? And also, we should rename `Cell` -> `WitnessCell`.